Monsoon/HVPM.py

```diff
@@ -78,7 +78,7 @@ def amps_from_raw(self,raw):
         return result
 
     def raw_from_amps(self,value):
-        result = 65535 - 0x0F00 * (value / 65535) + 0x0F00
+        result = (65535 * value / 15.625)
```

Comment:
Final correct line should be:

result = ((65535-0x0F00) * (value / 15.625)+0x0F00)

The hardware has a small negative offset voltage in the comparator circuit, so the 0x0F00 term is to correct for that.